### ISSUE TITLE
[ Latest Posts ] Option to always show the [Read more ...] link

### ISSUE THREAD

**Issue by draganescu** [WordPress/gutenberg#22269]
**Is your feature request related to a problem? Please describe.**
In #20313 ( [comment](https://github.com/WordPress/gutenberg/pull/20313#issuecomment-621335706) ) there was a discussion to have the block always show the link, even if:

- the theme does not add it
- the excerpt is not trimmed

**Describe the solution you'd like**
An option in the Block Inspector that shows only if `excerpt` is selected as the post content to be displayed. When this option is set, the link always is displayed after the excerpt.

cc @raQai

**Comment by jasomdotnet** [comment]
Also option to add custom class to this button would be useful.

**Comment by kathrynwp** [comment]
I agree that this would be a very good enhancement, particularly as I noticed that in a theme that does **not** display the "Read more" link on the front end – Twenty Twenty – the "Read more" link is still visible in the editor.

This creates confusion for the user, as the editor view does not match the front end.

###  Editor view example showing "Read more":

![Edit_Page_‹_Global_Styles_Demo_—_WordPress](https://user-images.githubusercontent.com/7662449/103691287-ece94800-4f63-11eb-901a-a4d60d50a708.jpg)

###  Front-end view of the same block

![Latest_Posts_Block_–_Global_Styles_Demo](https://user-images.githubusercontent.com/7662449/103691463-33d73d80-4f64-11eb-837f-a30852dcb228.jpg)

**Comment by thune** [comment]
I agree. This is missing and I have to use an external block to show posts with Read more. I like to use the blocks included in Wordpress. More futureproof :-)

**Comment by lefavilla** [comment]
+1. I would love to see this feature. I initally thought its a bug that read more is missing.

**Comment by draganescu** [comment]
Should we, now that WordPress 5.9 is out there, migrate Latest Posts to be based on a Query Loop block? Adding more features to it seems to be pointless as most of what this does the other one does also, possibly better.

**Comment by ghorivipul97** [comment]
Hello @draganescu 
I'm intrested to work on this issues. if is open then assign to me

**Comment by openmindculture** [comment]
Is this issue on a roadmap for 2025? If it isn't, what's the recommended best practice to show readmore links in the latests posts? My current workaround: add a custom excerpt to each post, and add a readmore link using HTML markup, e.g. `<a href=`... that works, although it's neither documented nor user-friendly for non-tech-savvy content editors.

**Comment by kathrynwp** [comment]
@openmindculture I'll let someone else respond to the roadmap question because I'm not sure about that, but have you tried just using the Query Loop block instead? I agree you shouldn't have to be adding Read More links manually with HTML! (Also not sure the Latest Posts block is being developed any longer, but again, I'll let someone else confirm that.)
